tgui/packages/tgui/interfaces/DecalPainter.tsx

```diff
@@ -0,0 +1,131 @@
+import { useBackend } from '../backend';
+import { Button, ColorBox, Flex, Section } from '../components';
+import { Window } from '../layouts';
+
+type Decal = {
+  name: string,
```

Comment:
Types should be delimited by `;`, might be useful to format code with prettier (using vscode).

Reply:
Fixed via prettier